test/integration/spec/remotetracks.js

```diff
@@ -109,7 +109,7 @@ describe('RemoteVideoTrack', function() {
 
     // eslint-disable-next-line no-warning-comments
     // TODO: enable these tests when track_priority MSP is available in prod
-    if (defaults.topology !== 'peer-to-peer' && defaults.environment === 'stage') {
+    if (defaults.topology !== 'peer-to-peer' && (defaults.environment === 'stage' || defaults.environment === 'dev')) {
```

Comment:
@makarandp0 
```suggestion
    if (defaults.topology !== 'peer-to-peer' && defaults.environment ! == 'prod') {
```
Does this not work?

Reply:
yes, that does not work because defaults.environment is `undefined` on Travis, and I think it defaults to prod.